### PR TITLE
Improve attribute handling for let_var_whitespace

### LINKED ISSUE #1770: let_var_whitespace should not trigger for @objc
I like the new `let_var_whitespace ` rule but I think it shouldn't trigger for `@objc` because it applies to that member. E.g. the following snippet triggers a violation:

```
    @objc(status)
    var statusObjc: String {
//...
```

What do you think?

### PR DESCRIPTION
**Pull request by Uncommon**
Fixes #1770

### CODE REVIEW

**Comment by marcelofabri** [comment]
I'm not sure this is the correct fix. You can see in https://github.com/realm/SwiftLint/pull/1776#issuecomment-322221705 that some fixed violations should actually be triggered.

**Comment by Uncommon** [comment]
Hey, good catch. I'll work on that. This means that the handling of `@available` wasn't quite right either.

**Comment by Uncommon** [comment]
I'll see how the CI goes and then rebase to fix the merge conflict.

File: CHANGELOG.md [hunk 1]
```diff
@@ -44,6 +44,11 @@
   [Hossam Ghareeb](https://github.com/hossamghareeb)
   [#1763](https://github.com/realm/SwiftLint/issues/1763)
 
+* Fix handling of attributes (`lazy`, `objc`, etc.) for the `let_var_whitespace`
+  rule.
```
Comment (ornithocoder) [inline CHANGELOG.md]: Hi @Uncommon. Don't forget the two trailing spaces after the description. More info [here](https://github.com/realm/SwiftLint/blob/master/CONTRIBUTING.md#tracking-changes)

Comment (Uncommon) [inline CHANGELOG.md]: Fixed and rebased.

**Comment by Uncommon** [comment]
Turns out it was my fault the tests were taking too long. Fixed.

**Comment by Uncommon** [comment]
Still needs work. It doesn't recognize attributes with underscores, or multiple lines of attributes before a declaration.

**Comment by Uncommon** [comment]
I looked through a bunch of the OS violation changes, and they look good to me.

**Comment by Uncommon** [comment]
Any more comments on this?

**Comment by marcelofabri** [comment]
Can you rebase this? It looks like it also fix #1812, so could you also add that as a non-triggering example and add the issue number to the changelog entry? Thanks!

**Comment by Uncommon** [comment]
Rebased and updated.

**Pull request merged**